demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -233,32 +271,34 @@ def copy_dir_yml(self, dir_path, bundle):
         if scan_files:
             print(f"\nStarting process for {dir_path}")
         for path in scan_files:
-            new_file_path = os.path.join(bundle, os.path.basename(path))
-            if len(os.path.basename(path)) >= self.file_name_max_size:
-                self.long_file_names.append(path)
-
-            ryaml = YAML()
-            ryaml.allow_duplicate_keys = True
-            with io.open(path, mode='r', encoding='utf-8') as file_:
-                yml_info = ryaml.load(file_)
-            ver = yml_info.get('fromversion', '0')
-            updated_yml_info = self.add_from_version_to_yml(yml_content=yml_info, save_yml=False)
-            if updated_yml_info:
-                yml_info = updated_yml_info
-
-            process_message = f' - processing: {path}'
-            if ver != '0' and ver != '':
-                process_message += f' - current fromversion: {ver}'
-            print(process_message)
-            if dir_name in ['Playbooks', 'TestPlaybooks']:
-                # in TestPlaybook dir we might have scripts - all should go to test_bundle
-                if dir_name == 'TestPlaybooks' and os.path.basename(path).startswith('script-'):
+            if self.check_from_version(path):
```

Comment:
will minimize indentantion levels (same for all uses).
```suggestion
            if not self.check_from_version(path):
                continue
```

Reply:
@roysagi you ignored this comment

Reply:
missed it. done now